### PR TITLE
Introduce --project flag to init command

### PR DESCRIPTION
**Pull request by azinneera**

### CODE REVIEW

File: components/cli/internal/test/bal_executor.go [hunk 1]
```diff
@@ -109,7 +109,7 @@ func (balExecutor *MockBalExecutor) Test(fileName string, args []string, envVars
 }
 
 // Build mocks execution of ballerina run for cellery init on an executable bal file.
-func (balExecutor *MockBalExecutor) Init(projectName string) error {
+func (balExecutor *MockBalExecutor) Init(workingDir string, projectName string, moduleName string) error {
```
Comment (MadhukaHarith92) [inline components/cli/internal/test/bal_executor.go]: Method parameters can be rewritten as "workingDir, projectName, moduleName string"

Comment (azinneera) [reply components/cli/internal/test/bal_executor.go]: Fixed in 8f88566

File: components/cli/pkg/commands/project/init.go [hunk 2]
```diff
@@ -21,17 +21,16 @@ package project
...
 	"strings"
-
-	"cellery.io/cellery/components/cli/cli"
+	"time"
 
 	"github.com/fatih/color"
 	"github.com/oxequa/interact"
 
+	"cellery.io/cellery/components/cli/cli"
+
```
Comment (MadhukaHarith92) [inline components/cli/pkg/commands/project/init.go]: Code formatting.

File: components/cli/pkg/commands/project/init_test.go [hunk 3]
```diff
@@ -54,7 +55,7 @@ func TestRunInit(t *testing.T) {
 	}
 	for _, tst := range tests {
 		t.Run(tst.name, func(t *testing.T) {
-			err := RunInit(mockCli, tst.project, "")
+			err := RunInit(mockCli, tst.project, tst.isProject)
```
Comment (MadhukaHarith92) [inline components/cli/pkg/commands/project/init_test.go]: Better if other scenarios in init command could be covered. i.e "initTest" function etc.

Comment (azinneera) [reply components/cli/pkg/commands/project/init_test.go]: Will be added by another PR. Created #990 to track the progress.

**Pull request merged**